packages/vow/src/vow-utils.js

```diff
@@ -38,3 +38,14 @@ export const getVowPayload = specimen => {
   return vow.payload;
 };
 harden(getVowPayload);
+
+export const vowishKey = k => {
```

Comment:
Should probably find a better name

Reply:
All clear names I came up with were too long. Suggestions appreciated.

At least it is now commented and typed.

Reply:
Having filed https://github.com/Agoric/agoric-sdk/issues/9302 , resolving here.